### DISCUSSION TITLE
Search Bar Broken on Codazon Infinit Theme

### DISCUSSION THREAD

**Discussion by KhaledBinAmir** [Smile-SA/elasticsuite#2170] [Q&A]
As you can see the screenshots, after installing Elasticsuite, the search bar is broken. Everything else is working fantastic, I have been trying this since long time but no luck.

Installed theme is [**Codazon Infinit**](https://themeforest.net/item/infinit-magento-2-magento-1-theme-multipurpose-responsive-theme/22432458)
URL [**mke.com.bd**](https://www.mke.com.bd/)

I am not a hardcore developer, can anyone help me if it can be fixed using custom CSS or any other tricks?

TIA!
![image](https://user-images.githubusercontent.com/30042185/121695118-91138400-caec-11eb-91c4-afe128e87722.png)
![image](https://user-images.githubusercontent.com/30042185/121695280-bc966e80-caec-11eb-8d9f-2a31c280f92e.png)

**Comment by romainruaud** [comment]
Hello @hiimkhaled 

we do not provide such kind of support for third-party extensions as part of our Open Source support policy here on Github.

I close it since we'll not provide any fix. You should ask your agency to fix this. However, I'm migrating this issue to "Discussions" so that it can remains open and hopefully somebody in the community might help you.

Best regards

**Comment by bve-9384** [comment]
I had a similar experience with a different theme and in my case the issue was that the theme simply redefined the top.search block.
Maybe not the best solution but I just modified the theme to use the top.search definition for elasticsuite instead:

```
<block class="Smile\ElasticsuiteCore\Block\Search\Form\Autocomplete" name="top.search" as="topSearch" template="Smile_ElasticsuiteCore::search/form.mini.phtml">
    <arguments>
        <argument name="rendererList" xsi:type="array">
            <item name="term" xsi:type="array">
                <item name="title" xsi:type="string" translate="true">Search terms</item>
                <item name="template" xsi:type="string">Smile_ElasticsuiteCore/autocomplete/term</item>
            </item>
            <item name="product" xsi:type="array">
                <item name="title" xsi:type="string" translate="true">Products</item>
                <item name="template" xsi:type="string">Smile_ElasticsuiteCatalog/autocomplete/product</item>
            </item>
            <item name="category" xsi:type="array">
                <item name="title" xsi:type="string" translate="true">Categories</item>
                <item name="template" xsi:type="string">Smile_ElasticsuiteCatalog/autocomplete/category</item>
            </item>
            <item name="product_attribute" xsi:type="array">
                <item name="title" xsi:type="string" translate="true">Attributes</item>
                <item name="template" xsi:type="string">Smile_ElasticsuiteCatalog/autocomplete/product-attribute</item>
                <item name="titleRenderer" xsi:type="string">Smile_ElasticsuiteCatalog/js/autocomplete/product-attribute</item>
            </item>
        </argument>
    </arguments>
</block>
```


Just deleting the theme's version of top.search did not work for me, did not spend more time on a better a solution...
Using Magento 2.3.7 and Elasticsuite 2.9
